scrapers/Nympho.yml

```diff
@@ -20,22 +20,15 @@ xPathScrapers:
         Name: $info//h4[@class="models"]/a
       Studio:
         Name:
-          selector: //a[contains(@class,"navbar-brand")]/@href
-          postProcess:
-            - replace:
-                - regex: .*tour\.(\w+).*
-                  with: $1
-            - map:
-                nympho: Nympho
-                trueanal: TrueAnal
-                swallowed: Swallowed
-                allanal: AllAnal
+          selector: //a[contains(@class,"navbar-brand")]/img/@title
       Date:
         selector: $info//span[contains(@class,"date") and not(contains(@class,"mobile"))]
         postProcess:
           - replace:
               - regex: (\d+)(?:[dhnrst]+)\s+(.+)
                 with: "$1 $2"
           - parseDate: 2 Jan 2006
-      Image: //video[contains(@id,"ypp-player")]/@poster|//div[@id="main-player"]/@data-screenshot
-# Last Updated December 06, 2021
+      Image: //video[contains(@id,"ypp-player")]/@poster|//div[@id="main-player"]/@data-screenshot|//meta[@property="og:image"]/@content
+driver:
+  useCDP: true                     
```

Comment:
I didnt notice that before...
Why was cdp added here?

Reply:
Have removed it, Think I had it in there as sometimes it would block you. I think it looks more like a cookie thing.